website/webpack.config.js

```diff
@@ -89,6 +80,12 @@ const COMMON_CONFIG = {
       {
         test: /\.(svg|ico|gif|jpe?g|png)$/,
         loader: 'file-loader?name=[name].[ext]'
+      },
+      // for compiling apache-arrow ESM module
+      {
+        test: /\.mjs$/,
+        include: /node_modules\/apache-arrow/,
+        type: 'javascript/auto'
```

Comment:
this makes sure apache arrow files are imported correctly. copied from examples/webpack.config.local.js